lib/rabbitmq/cli/diagnostics/commands/cipher_suites_command.ex

```diff
@@ -0,0 +1,42 @@
+## The contents of this file are subject to the Mozilla Public License
+## Version 1.1 (the "License"); you may not use this file except in
+## compliance with the License. You may obtain a copy of the License
+## at http://www.mozilla.org/MPL/
+##
+## Software distributed under the License is distributed on an "AS IS"
+## basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See
+## the License for the specific language governing rights and
+## limitations under the License.
+##
+## The Original Code is RabbitMQ.
+##
+## The Initial Developer of the Original Code is GoPivotal, Inc.
+## Copyright (c) 2007-2016 Pivotal Software, Inc.  All rights reserved.
+
+
+defmodule RabbitMQ.CLI.Diagnostics.Commands.CipherSuitesCommand do
```

Comment:
Not directly related to this PR: locating this command as part of the `Diagnostics` namespace makes it appear in the help blurb *after* commands that explicitly specify the scope (e.g. https://github.com/rabbitmq/rabbitmq-cli/blob/master/lib/rabbitmq/cli/ctl/commands/cluster_status_command.ex#L27).

```
$ escript/rabbitmq-diagnostics              

Command '' not found. 
Usage:
rabbitmq-diagnostics [-n <node>] [-q] <command> [<command options>]
...
Commands:
    cluster_status
    environment
    help <command>
    list_bindings [-p <vhost>] [<bindinginfoitem> ...]
    list_channels [<channelinfoitem> ...]
    list_connections [<connectioninfoitem> ...]
    list_consumers [-p vhost] [<consumerinfoitem> ...]
    list_exchanges [-p <vhost>] [<exchangeinfoitem> ...]
    list_operator_policies [-p <vhost>]
    list_parameters [-p <vhost>]
    list_permissions [-p <vhost>]
    list_policies [-p <vhost>]
    list_queues [-p <vhost>] [--online] [--offline] [<queueinfoitem> ...]
    list_user_permissions <username>
    list_users
    list_vhosts [<vhostinfoitem> ...]
    node_health_check
    report
    status
    wait <pid_file>
    cipher_suites [--openssl-format]
<bindinginfoitem> must be a member of the list [source_name, source_kind, destination_name, destination_kind, routing_key, arguments].
...
```

It'd make more sense to have the commands listed in alphabetical order. It's a small usability problem, but I think it's worth investigating in another issue.